### PR TITLE
chore: publish v1.0.0 🚀

### PR DESCRIPTION
**Pull request by dshukertjr**
Also included the primary key breaking change discussed [here](https://github.com/supabase-community/supabase-flutter/issues/218#issuecomment-1279038003).

```dart
// Old
supabase.from('table').stream(['id']);

// New
supabase.from('table').stream(primaryKey: ['id']);
```

### CODE REVIEW

File: lib/src/supabase_query_builder.dart [hunk 1]
```diff
@@ -37,14 +37,15 @@ class SupabaseQueryBuilder extends PostgrestQueryBuilder {
   /// ```dart
   /// supabase.from('chats:room_id=eq.123').stream(['my_primary_key']).order('created_at').limit(20).execute().listen(_onChatsReceived);
   /// ```
-  SupabaseStreamBuilder stream(List<String> uniqueColumns) {
+  SupabaseStreamBuilder stream({required List<String> primaryKey}) {
+    assert(primaryKey.isNotEmpty, 'Please specify parimary key(s).');
```
Comment (Vinzent03) [inline lib/src/supabase_query_builder.dart:41]: We should change the name and typo here.

**Review by Vinzent03 — CHANGES_REQUESTED** [review]

File: lib/src/supabase_query_builder.dart (hunk 1 above)
Comment (dshukertjr) [reply lib/src/supabase_query_builder.dart]: Thanks for picking up on the typo! What are you suggesting about the name change?

Comment (Vinzent03) [reply lib/src/supabase_query_builder.dart]: There is again the plural of primary key. I would remove ` (s)`

Comment (dshukertjr) [reply lib/src/supabase_query_builder.dart]: @Vinzent03 Sure, how about right now?

**Review by Vinzent03 — APPROVED** [review]
LGTM!

**Comment by dshukertjr** [comment]
Thanks for the quick review! Exciting times! Now I will work on the supabase-flutter rework, and we can finally release v1!

**Pull request merged**